css/properties/overflow-y.json

```diff
@@ -100,7 +100,7 @@
                 },
                 {
                   "alternative_name": "overlay",
-                  "version_added": "≤13.1"
+                  "version_added": "6"
```

Comment:
Same as: https://github.com/mdn/browser-compat-data/pull/27186#discussion_r2193136375

Reply:
```suggestion
                  "version_added": "6",
                  "notes": "Before Safari 12.1, `overlay` only had an effect on legacy scrollbars on macOS. Since Safari 12.1, it is parsed as `auto`."
```